src/main/java/com/dev/cinema/model/MovieSession.java

```diff
@@ -9,15 +9,13 @@
 import javax.persistence.Table;
 
 @Entity
-@Table(name = "movie session")
+@Table(name = "movie_session")
```

Comment:
```suggestion
@Table(name = "movie_sessions")
```
if you choose to name table for User `users` - let's have plural for **all** other tables as well

Reply:
I'll change users table to user. Had problems with sql db connection and queries in some classes, so tried figure out what's wrong and where.